### PR TITLE
matching to arm64

### PR DESCRIPTION
**Pull request by tuvia-akeyless**

### CODE REVIEW

**Review by liorhasson — APPROVED** [review]

File: makefile [hunk 1]
```diff
@@ -23,15 +21,22 @@ testacc: fmtcheck
 	TF_ACC=1 go test $(TEST) -v -count 1 -timeout 120m
 
 build-linux:
-	GOOS=linux GOARCH=amd64 go build -o ${BINARY}
+	GOOS=linux OS_ARCH=darwin_amd64 GOARCH=amd64 go build -o ${BINARY}
```
Comment (renanaAkeyless) [inline makefile:24]: linux with darwin_amd64 ??

File: makefile [hunk 2]
```diff
@@ -23,15 +21,22 @@ testacc: fmtcheck
...
 	mkdir -p ~/.terraform.d/plugins/${SOURCEHOST}/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/linux_amd64
 	mv ${BINARY} ~/.terraform.d/plugins/${SOURCEHOST}/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/linux_amd64
 
 install-darwin: build-darwin
 	mkdir -p ~/.terraform.d/plugins/${SOURCEHOST}/${NAMESPACE}/${NAME}/${VERSION}/darwin_amd64
-	mv ${BINARY} ~/.terraform.d/plugins/${SOURCEHOST}/${NAMESPACE}/${NAME}/${VERSION}/darwin_amd64
\ No newline at end of file
+	mv ${BINARY} ~/.terraform.d/plugins/${SOURCEHOST}/${NAMESPACE}/${NAME}/${VERSION}/darwin_amd64
+
+install-darwin-m1: build-darwin
```
Comment (renanaAkeyless) [inline makefile:40]: build-darwin-m1:

File: makefile (hunk 1 above)
Comment (tuvia-akeyless) [reply makefile:24]: V

File: makefile (hunk 2 above)
Comment (tuvia-akeyless) [reply makefile:40]: V

**Review by rani-sharim — APPROVED** [review]

**Review by renanaAkeyless — APPROVED** [review]

**Pull request merged**